opensrp-giz-malawi/src/main/java/org/smartregister/giz/activity/ReportTypeListRegisterActivity.java

```diff
@@ -0,0 +1,68 @@
+package org.smartregister.giz.activity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import org.smartregister.domain.FetchStatus;
+import org.smartregister.giz.R;
+import org.smartregister.giz.model.ReportGroupingModel;
+
+import java.util.ArrayList;
+
+public class ReportTypeListRegisterActivity extends ReportRegisterActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
```

Comment:
There is no need to override this method.

Reply:
Rather override this https://github.com/OpenSRP/opensrp-client-giz-malawi/pull/358#discussion_r570155823

Reply:
Done